bindings/wasm/bindings.cpp

```diff
@@ -72,6 +73,13 @@ Manifold Transform(Manifold& manifold, std::vector<float>& mat) {
   return manifold.Transform(matrix);
 }
 
+Manifold Warp(Manifold& manifold, void* funcPtr) {
+  void (*f)(glm::vec3&) = reinterpret_cast<void (*)(glm::vec3&)>(funcPtr);
```

Comment:
This part feels the most wrong to me. I'm slightly hazy on the distinction between a reference and a pointer, but I feel like the JS wants a pointer and the C++ is giving a reference to the `glm::vec3` and I'm not really sure how to resolve that. And I'm not even sure that's the error I'm running into.

Reply:
Well, it's working now, so apparently that distinction is pretty moot. I'm honestly amazed this works.

Reply:
Well, references are basically implemented as pointers under the hood, but with a bit more restrictions (cannot be null, cannot have `T&&&&&&` while you can have `T********`, for example).